From 8206e03abfe80c4801d3573bd43252fb37bb69be Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Tue, 30 Aug 2016 14:33:18 -0400
Subject: [PATCH] rtl8xxxu: Clear SYS_FUNC_UPLL during power up on 8188eu

The vendor driver doesn't set this bit during BB config, so avoid
setting it here too.

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---
 drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
+++ b/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
@@ -561,6 +561,7 @@ static void rtl8188eu_init_phy_bb(struct
 	val16 = rtl8xxxu_read16(priv, REG_SYS_FUNC);
 	val16 |= (SYS_FUNC_USBA | SYS_FUNC_USBD |
 		  SYS_FUNC_BB_GLB_RSTN | SYS_FUNC_BBRSTB);
+	val16 &= ~SYS_FUNC_UPLL;
 	rtl8xxxu_write16(priv, REG_SYS_FUNC, val16);
 
 	rtl8xxxu_init_phy_regs(priv, rtl8188eu_phy_init_table);
